docs/VMR-Feature-Band-Support-Work-Plan.md

```diff
@@ -9,9 +9,16 @@ Document convention:
 
 ### Branch Management
 
-The 2xx branch sources should accurately reflect what gets produced in the output. Since the shared component are only being produced in the 1xx branch, those associated repos will be excluded from the 2xx branch upon creation.
+The 2xx branch sources should accurately reflect what gets produced in the output. Since the shared component are only being produced in the 1xx branch, those associated repos will be excluded from the 2xx branch upon creation. The benefits of of removing the source include the following:
+* Avoids confusion about which source is relevant in the production of the build
+* Avoids the need to determine if or when repo sources should be updated with the latest from the 1xx branch.
+* Avoids any compliance issues with the code getting stale if it hasn't been updated (e.g. CodeQL alerts).
 
-There should be tooling to help with this exclusion process for the VMR maintainers. Similarly, the associated projects in the `repo-projects` directory should be deleted along with updating the appropriate `RepositoryReference` usages.
+One exception is the Arcade repo which will be present in all branches. It's not technically required to exist in the 2xx branch in all cases. But there have historically been cases where Arcade changes are necessary in order to support new SDK features not in the 1xx branch. To account for those cases, it'll be a smoother process to simply always have Arcade present in those branches. There isn't an impact in the output of the build since Arcade isn't a shipping component.
+
+Rather than deleting the associated projects in the `repo-projects` directory, it may make sense to keep those and trim the dependency graph of `RepositoryReferences` using a condition based on whether it's the 1xx branch or not.
+
+There should be tooling to help with this exclusion process for the VMR maintainers. This can potentially be integrated into the broader workflow that is necessary when branching such as branding updates and pipeline changes. The tooling could potentially make use of the dependency graph described by the `repo-projects` directory to determine which repos need to be removed.
```

Comment:
If there should be tooling for snapping bands, we should possibly describe what the snap looks like in detail?
I know something like this is already in the original design documents but maybe we know more now.